src/cljs/main/broadfcui/page/billing/billing_management.cljs

```diff
@@ -14,7 +14,7 @@
     [broadfcui.page.billing.create-project :refer [CreateBillingProjectDialog]]
     [broadfcui.page.billing.manage-project :refer [BillingProjectManagementPage]]
     [broadfcui.page.workspace.monitor.common :as moncommon]
-    [broadfcui.utils :as utils]
+    [broadfcui.utils :as u]
```

Comment:
I'm not in love with this shortening, but then again, I'm just this guy, you know?